delfin/drivers/fake_storage/__init__.py

```diff
@@ -153,9 +157,9 @@ def list_storage_pools(self, ctx):
         for idx in range(rd_pools_count):
             total, used, free = self._get_random_capacity()
             p = {
-                "name": "fake_pool_" + str(idx),
+                "name": "storagePool_" + str(idx),
                 "storage_id": self.storage_id,
-                "native_storage_pool_id": "fake_original_id_" + str(idx),
+                "native_storage_pool_id": "storagePool_" + str(idx),
```

Comment:
Convention looks fine, how about using "native_" prefix for native ids just to differentiate it from name

Reply:
IMO which will be again confusing.. because in metric we keep resource_id as a common label for all resources..  It is supposed to be the native_id .. in order to avoid confusion lets keep resource name and native_id same..

Reply:
LGTM